chainer/gradient_check.py

```diff
@@ -21,6 +22,34 @@ def _copy_arrays(xs):
     return [xp.array(x, order='C', dtype=numpy.float64, copy=True) for x in xs]
 
 
+def _ones_like(arr):
+    device = cuda.get_device_from_array(arr)
+    xp = backend.get_array_module(arr)
+    with device:
+        return xp.ones_like(arr)
+
+
+def _check_outputs_and_grad_outputs(outputs, grad_outputs):
+    if len(outputs) != len(grad_outputs):
+        raise ValueError(
+            'Output gradients must contain equally as many elements as '
+            'the number of output elements.\n'
+            'Outputs         : {}\n'
+            'Output gradients: {}'.format(
+                ', '.join(str(y.shape) for y in outputs),
+                ', '.join(str(None if gy is None else gy.shape)
+                          for gy in grad_outputs)))
```

Comment:
Why are the `shape`s printed on this error?

Reply:
I thought it would be helpful for those who read this message to spot the cause.

Reply:
I got your intention.  Could you make the error message clearer in that the shapes are printed?

Reply:
Fixed the messages.